kube/minkube/README.md

```diff
@@ -137,15 +137,15 @@ kubectly apply -f api-deployment.yaml
 
 ## Cleaning up everything afterwards
 
-To clean up everything simply use the ```clean-all.sh``` script provided in the root of the kube/minikube/hack folder by running
+To clean up everything simply use the ```clean-all.sh``` script provided in the root of the kube/minikube/scripts folder by running
```

Comment:
@JenySadadia how about calling clean-all.sh tear.sh instead.

Reply:
`delete-resources.sh` looks more suitable to me.